profile/arc_cwl_ro_crate.md

```diff
@@ -23,11 +23,14 @@ To continue staying consisten with the [ISA RO-Crate Profile](https://github.com
 
 ```mermaid
 flowchart TD
-        A["File\nSoftwareSourceCode\nComputationalWorkflow"] -- "input\noutput" --> B["FormalParameter"]
-        A -- "instrument" --> C["CreateAction"]
+        A["File\nSoftwareSourceCode\nComputationalWorkflow\nLabProtocol"] -- "input\noutput" --> B["FormalParameter"]
```

Comment:
In my opinion, this reads like a list of options for the type. Maybe annotate it as `[File,SoftwareSourceCode,...]`? This holds for all multi types of course.